elastic/kibana
# [RAM] Show Alerting rule JSON for API requests

Resolves https://github.com/elastic/kibana/issues/178235

## Summary

This PR adds a new button the rule flyout that will show the rule API request so the user can use it later on in API requests.

**Create**

<img width="611" alt="Screen Shot 2024-04-04 at 12 55 24 PM" src="https://github.com/elastic/kibana/assets/109488926/882dc34e-b1e4-40f1-ab8e-ad45f361dced">
<img width="788" alt="Screen Shot 2024-04-04 at 12 55 32 PM" src="https://github.com/elastic/kibana/assets/109488926/f2313963-646d-496d-bd63-3bcc3b985e5f">

**Edit**

<img width="610" alt="Screen Shot 2024-04-04 at 12 55 45 PM" src="https://github.com/elastic/kibana/assets/109488926/57343b3c-b67c-4da6-9649-9de9eb98f3bc">
<img width="794" alt="Screen Shot 2024-04-04 at 12 54 40 PM" src="https://github.com/elastic/kibana/assets/109488926/512d6a81-424d-435c-81b5-c8a873d92e61">


### Checklist



### To verify
1. Use the rule form to create a new rule
2. Before saving click the `Show request` button and copy the request
3. Go to [dev tools](http://localhost:5601/app/dev_tools#/console) and run the request, verify that the rule is created properly
4. Go back to the [rule page](http://localhost:5601/app/management/insightsAndAlerting/triggersActions/rules) and edit your newly created rule
5. Before saving the edits repeat step 2
6. Go back to [dev tools](http://localhost:5601/app/dev_tools#/console) and run the edit request, verify that the rule is properly updated
